docs/pages/starter-kit/index.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Starter Kit'
+---
+
+# Starter kit
+
+The starter kit is an example of an application running on the Liteflow infrastructure to showcase the uses of the infrastructure and/or be used as a starter kit to launch a product fast.
+
+This application is fork-able on this repository: https://github.com/liteflow-labs/marketplace-template.
+
+## Features included
+
+The Marketplace template includes the following features:
+
+- Open collections (User-generated content)
+  - Mint ERC-721
+  - Mint ERC-1155
+  - Creator's royalties support
+- Featured NFTs on the home page
+- Search system
+- Explore NFTs with filtering and sorting systems
+- Native currency support
```

Comment:
```suggestion
- Native currency and ERC20 support
```

Reply:
updated the marketplace template repo as well